website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -129,3 +129,26 @@ The SideNav should always be positioned on the left side of the viewport, occupy
 On smaller viewports, the SideNav should collapse to maximize the available real estate on tablet and mobile devices. By tapping the menu icon, users can expand and access the full menu when needed.
 
 ![Responsive side-nav](/assets/components/side-nav/sidenav-position-and-responsive.png)
+
+## Collapse functionality
+
+If the `isCollapsible` property is set to true, a collapse toggle button will be exposed to the end-user allowing them to manually expand and collapse the component. 
+
+![SideNav collapse function](/assets/components/side-nav/sidenav-collapse-interaction.png)
+
+On smaller viewports the SideNav will be rendered in its collapsed state **by default** and will overlay the main page content it its expanded state.
+
+![SideNav overlay on smaller viewports](/assets/components/side-nav/sidenav-overlay-small-viewport.png)
```

Comment:
Please double check (unless you haven't done it yet) that the menu items can be publicly shared (in this as well as in the other images).

Notice: I am going to set the review to blocked, so we don't accidentally merge it without checking this. Once we're sure, I'll change it to approved.

Reply:
👍 \I'll check with the ambassador group

Reply:
I made one small tweak on the Vagrant menu item which should satisfy questions around unreleased/sensitive information.

Reply:
Thanks for checking 🙏